src/python/pants/task/fmt_task_mixin.py

```diff
@@ -37,7 +18,6 @@ def act_transitively(self):
     def determine_if_skipped(self, *, formatter_subsystem: Subsystem) -> bool:
         # TODO: generalize this to work with every formatter, not only scalafix.
```

Comment:
Maybe now it's not in the same file anymore, I would add a sentence to this comment to say: "when doing this, remember to change the help description for `--only` in `rules/core/fmt.py`." I could see us forgetting that otherwise.